CMakeLists.txt

```diff
@@ -28,6 +29,22 @@ include(CheckLanguage)
 enable_language(CUDA)
 check_language(CUDA)
 
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+configure_file(
+  "${CMAKE_SOURCE_DIR}/src/hykkt_defs.hpp.in"
+  "${CMAKE_BINARY_DIR}/hykkt_defs.hpp")
+include_directories(${CMAKE_BINARY_DIR})
```

Comment:
I think we should avoid blanket include_directories, but I don't think we follow that in ExaGO or HiOp, so no big deal